chain/jsonrpc/res/last_blocks.html

```diff
@@ -2,9 +2,24 @@
 
 <head>
     <style>
+        .explanation {
+            color: gray;
+        }
+
+        .missed-blocks,
+        .missed-chunks {
+            margin-top: 10px;
+            margin-bottom: 10px;
+        }
+
+        button {
+            margin-right: 8px;
+        }
+
         table {
             width: 100%;
             border-collapse: collapse;
+            margin-top: 10px;
```

Comment:
we should probably move the CSS to a separate file too (but in a separate PR) - especially for the common things like tables etc.

Reply:
Ah yes. Would be nice to pull a common js out too since we need to display the banner for the server info for another issue.